app/views/shares/index.html.erb

```diff
@@ -6,7 +6,7 @@
     <div class="handle"><%= current_user.name %></div>
     <%= link_to share.title, share_path(share) %>
     <p>
-      <%= linkify_hashtags(share.body) %>
+      <%= linkify_hashtags.markdown(share.body) %>
```

Comment:
Does this actually work?  I would expect the code to look more like:

```
linkify_hashtags(markdown(share.body))
```

Unless there's some way the hashtag gem and redcarpet interact that I don't understand.